### PR TITLE
feat: add multilanguage names for connectivity environment

### PR DESCRIPTION
**Pull request by srieding**
refs: withthegrid/platform#1255

### CODE REVIEW

File: src/models/command-type.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import Joi from 'joi';
 
 import { schema as baseFieldConfigurationSchema, BaseFieldConfiguration } from './fields/base-field-configuration';
+import { schema as stringOrTranslationsSchema, StringOrTranslations } from './string-or-translations';
 
 const schema = (apiVersion: number): Joi.ObjectSchema => Joi.object().keys({
   hashId: Joi.string().required().example('x18a92'),
-  name: Joi.string().required().example('Measurement cycle'),
+  name: stringOrTranslationsSchema.required().example('Measurement cycle'),
```
Comment (srieding) [inline src/models/command-type.ts:8]: This ends up in the get but also in the find route.

Comment (everhardt) [inline src/models/command-type.ts:8]: `name` should be string for apiVersion <= 6 (also applicable to other cases in this PR)

**Review by everhardt — CHANGES_REQUESTED** [review]

File: src/models/string-or-translations.ts [hunk 2]
```diff
@@ -10,7 +11,25 @@ const schema = Joi.alternatives(
 
 type StringOrTranslations = string | Translations;
 
+function getTranslatedString(
+  name: StringOrTranslations,
+  locale: Locale,
+): string {
+  if (name === null) {
```
[the remark this answers is not in the document]
Comment (everhardt) [reply src/models/string-or-translations.ts:18]: But you can't provide title as the `name` parameter for this function, right? My point is that `StringOrTranslations` does not contain `null`.

[the remark this answers is not in the document]
Comment (srieding) [reply src/models/string-or-translations.ts:18]: Hmm, that is technically correct though not enforced in the frontend code. So I ended up calling this with null and undefined from the frontend. I could of course make a separate function for that usecase

[the remark this answers is not in the document]
Comment (everhardt) [reply src/models/string-or-translations.ts:18]: Maybe the function overloading can help out here as well..

**Review by everhardt — APPROVED** [review]
Can be merged when https://github.com/withthegrid/platform/pull/1319/ is ready

**Pull request merged**